sql/2021/markup/attributes.sql

```diff
@@ -30,6 +31,7 @@ GROUP BY
   client,
   almanac_attribute_info.name
 ORDER BY
+  pct_ratio DESC,
```

Comment:
Is sorting by pct_ratio preferable to using `ROW_NUMBER()` and then filtering on it?
```
SELECT
  ..
  ROW_NUMBER() OVER (PARTITION BY client ORDER BY COUNT(0) DESC) AS pos
FROM
  ..
WHERE pos <= 100
```

Reply:
Using `ROW_NUMBER() OVER...` does have the advantage of being able to have exactly 100 mobile and 100 desktop results. However in reality that rarely matters as we normally only report on the top 5, or 10, or 15. So as long as you have enough values (100 or 200 is usually a good enough number) and usage is high enough on both mobile and desktop then it rarely matters.

And just ordering by `pct` is a heck of a lot simpler 😁